examples/experimental/i3s-17-and-debug/mesh-layer/simple-mesh-layer-vertex.glsl.js

```diff
@@ -11,7 +11,7 @@ in vec3 normals;
 in vec3 colors;
 in vec2 texCoords;
 in vec4 uvRegions;
-in vec3 pickingColors;
+in vec3 segmentationPickingColors;
```

Comment:
If we are using ifdefs instead of uniforms, we should probably put the declarations under ifdef as well.

Generally ifdefs can be slightly faster than using a uniform, however, defines have the problem that one path of the shader is not seen by the GLSL compiler so it is much easier to break the shader during maintenance unless we have test cases that test both paths.

Reply:
It is not clear for me how to put declaration of `segmentationPickingColors` inside `ifdef` block. I got an error when I try to put it inside.

Reply:
You can't move it inside the existing ifdef, It has to go on top. You wrap it in another ifdef
```suggestion
#ifdef ...
in vec3 segmentationPickingColors;
#endif
```